src/imageio.cs

```diff
@@ -901,6 +901,734 @@ interface CGImageProperties {
 		[Mac (10, 13), iOS (11,0), TV (11,0), Watch (4,0)]
 		[Field ("kCGImagePropertyFileContentsDictionary")]
 		NSString FileContentsDictionary { get; }
+		
+		[Mac (10, 9)][iOS (11,0)]
+		[Field ("kCGImagePropertyOpenEXRDictionary")]
+		NSString OpenEXRDictionary { get; }
+
+		[Mac (10, 13, 4)][iOS (11, 3)]
+		[Field ("kCGImagePropertyIPTCExtAboutCvTerm")]
+		NSString IPTCExtAboutCvTerm { get; }
```

Comment:
-> `Iptc`

Reply:
We have a _ton_ of IPTCs already and I was trying to stay consistent - https://github.com/xamarin/xamarin-macios/blob/master/src/imageio.cs#L52

Reply:
make sense, it looked like a new one :)